core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -1985,6 +1991,12 @@
       <trans-unit id="MSC.groupListingModule">
         <source>There is at least one user group granting permission to the front end module "Listing" and its "list_*" fields. Since they can contain arbitrary SQL queries, users of these groups implicitly have full control over the system!</source>
       </trans-unit>
+      <trans-unit id="MSC.groupUnfilteredHtml">
+        <source>There is at least one user group granting access to the unfiltered HTML element. Since the HTML code of that element does not get filtered, users of these groups implicitly have full control over the system!</source>
```

Comment:
```suggestion
        <source>There is at least one user group that grants access to the unfiltered HTML element. Since the HTML code of this element is not filtered, the users of these groups implicitly have full control over the system!</source>
```